### PR TITLE
fix README instructions around the dependencies 

### PR DESCRIPTION
**Pull request by afeld**
Fixes https://github.com/opencontrol/compliance-masonry/issues/186.

See commits.

/cc @cmc333333

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,13 +55,13 @@ Compliance Masonry is a command-line interface (CLI) that allows users to constr
       - ./standard-1.yaml
     dependencies:
       certifications: # An optional list of certifications stored remotely
-        - url: github.com/18F/LATO
+        - url: https://github.com/18F/LATO
```
Comment (pburkholder) [inline README.md]: https://github.com/18F/LATO is not a publicly accessible URL. Does LATO live somewhere we can access it?

File: README.md [hunk 2]
```diff
@@ -55,13 +55,13 @@ Compliance Masonry is a command-line interface (CLI) that allows users to constr
...
           revision: master
       systems:  # An optional list of repos that contain an opencontrol.yaml stored remotely
-        - url: github.com/18F/cg-compliance
+        - url: https://github.com/18F/cg-compliance
           revision: master
       standards:   # An optional list of remote repos containing standards info that contain an opencontrol.yaml
-        - url: github.com/18F/NIST-800-53
+        - url: https://github.com/18F/NIST-800-53
```
Comment (pburkholder) [inline README.md]: https://github.com/18F/NIST-800-53 also not accessible.

Comment (pburkholder) [inline README.md]: https://github.com/opencontrol/NIST-800-53-Standards perhaps?

**Comment by pburkholder** [comment]
Thanks for this. The README I think is still non-functioning without valid contents for this:

```
    components: # A list of paths to components written in the opencontrol format for more information view: https://github.com/opencontrol/schemas
       - ./component-1
     certifications: # An optional list of certifications for more information visit: https://github.com/opencontrol/schemas
       - ./cert-1.yaml
     standards: # An optional list of standards for more information visit: https://github.com/opencontrol/schemas
        - ./standard-1.yaml
```

but one step at a time.

File: README.md (hunk 1 above)
Comment (pburkholder) [inline README.md]: Ah, it seems to be here: https://github.com/18F/GSA-Certifications/blob/master/LATO.yaml

File: README.md (hunk 2 above)
Comment (jcscottiii) [inline README.md]: Good catch @pburkholder!

File: README.md (hunk 1 above)
Comment (jcscottiii) [inline README.md]: Yep! That's it

**Comment by afeld** [comment]
Updated!

**Pull request merged**